torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -83,6 +83,24 @@ std::pair<XLATensor, XLATensor> GetBinaryOperands(const at::Tensor& self,
   return std::pair<XLATensor, XLATensor>(self_tensor, other_tensor);
 }
 
+// The input is in format of {N, C, H, W} and the output will be {H, W}.
+std::vector<xla::int64> GetOutputSizeWithScale(
+    absl::Span<const xla::int64> input_size,
+    c10::optional<at::ArrayRef<double>> scale_factors,
```

Comment:
Even though pytorch passes those by value (these are refs, so their storage/copy is small), I think it is best to stick with the policy to pass optional values by `const&` as passing by value involves copies.